### DISCUSSION TITLE
Is there a way to register as a user with only a phone number without a password?

### DISCUSSION THREAD

**Discussion by hhamja** [supabase/supabase#8102] [Questions]
Is there a way to register as a user with only a phone number without a password? For user convenience, I would like to have users sign up only with their mobile phone number and OTP authentication without setting a password.

**Comment by Hallidayo** [comment]
Hi @CodeKinggggggg,

It may be worth looking at these mobile auth guides:

- [Phone Auth with Twilio](https://supabase.com/docs/guides/auth/auth-twilio)
- [Phone Auth with Vonage](https://supabase.com/docs/guides/auth/auth-vonage)
- [Phone Auth wth MessageBird](https://supabase.com/docs/guides/auth/auth-messagebird)

All of the above use OTP, You would just need to choose which one is right for your project.

**Reply by hhamja** [reply]
I've already seen the Twilio documentation and tried to do it. However, there are no examples of flutter in the documentation, so I can't proceed.
